src/assets/app.js

```diff
@@ -0,0 +1,62 @@
+import './bootstrap.js';
+/*
+ * Welcome to your app's main JavaScript file!
+ *
+ * This file will be included onto the page via the importmap() Twig function,
+ * which should already be in your base.html.twig.
+ */
+import './styles/app.css';
+
+console.log('aaaaa')
```

Comment:
These debug statements should ideally be removed from the code before deploying it somewhere